elliott/elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -188,7 +188,10 @@ def verify_bugs_advisory_type(self, non_flaw_bugs, advisory_id_map, advisory_bug
         bugs_by_type = categorize_bugs_by_type(non_flaw_bugs, advisory_id_map, permitted_bug_ids=permitted_bug_ids)
         for kind, advisory_id in advisory_id_map.items():
             actual = {b for b in advisory_bug_map[advisory_id] if b.is_ocp_bug()}
-            expected = bugs_by_type[kind]
+            expected = bugs_by_type.get(kind)
+            if not expected:
+                # Looks like impetus is not part of the ones we care about. Skipping bug attachment
+                continue
```

Comment:
```suggestion
            # If impetus is not present, assume no bugs need to be attached 
            expected = bugs_by_type.get(kind, set())
```

Reply:
> For "advance" operator advisory we will attach bugs to it
So for that we just need to add a line [here](https://github.com/openshift-eng/art-tools/blob/9331f7a357e88cda10f4c9f36dac5d51fa09ac06/elliott/elliottlib/cli/find_bugs_sweep_cli.py#L274) right, for `advance`.